pom.xml

```
@@ -71,6 +71,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <version.jackson>2.12.2</version.jackson>
+    <version.jackson-datatype>2.11.4</version.jackson-datatype>
```

Let's please synchronize versions of Jackson libs.

```suggestion
    <version.jackson>2.12.2</version.jackson>
```